hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -931,12 +931,28 @@ public FileStatus[] listStatus(final Path f) throws IOException {
           == PrefixMode.BLOB) {
         FileStatus renamePendingFileStatus
             = getAbfsStore().getRenamePendingFileStatus(result);
```

Comment:
Rename variable to renamePendingJSONStatus

Reply:
refactored.